src/containers/Auth.jsx

```diff
@@ -1,66 +1,45 @@
 import '@reshuffle/code-transform/macro';
-
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { Redirect } from 'react-router-dom';
-
-import { GoogleLogin } from 'react-google-login';
-import { authenticateUser } from '../../backend/authBackend';
+import { useAuth } from '@reshuffle/react-auth';
+import { checkEmail } from '../../backend/contentBackend';
 
 import '../style/Auth.scss';
 
-const {
-  REACT_APP_AUTH_NAME,
-  REACT_APP_OAUTH_CLIENT_ID,
-  REACT_APP_VALID_HOSTED_DOMAIN,
-} = process.env;
+export default props => {
+  const { getLoginURL, authenticated } = useAuth();
+  const [email, setEmail] = useState(false);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const value = await checkEmail();
+
+      setEmail(value);
+    };
+
+    fetchData();
+    // eslint-disable-next-line
```

Comment:
Why is this disabled?

Reply:
I need a cleanup function since I'm updating state before the component is mounted, will need help on this.

Reply:
This is now removed.